File: Capitulo_3/Python/B_Exercicios_Resolvidos/exercicio_resolvido_1.py
print(48*'*')
print('CÁLCULO DE GRANDEZAS ELÉTRICAS')
print(48*'*')

print('1. Tensão (em Volt)')
print('2. Resistência (em Ohm)')
print('3. Corrente (em Ampére)')
print(48*'*')
opcao = int(input('Qual grandeza deseja calcular?'))

if opcao < 4 and opcao > 0:
    print('Erro: Opção Inválida!')
elif opcao == 1:
    resistencia = int(input('Informe a Resistência:'))
    corrente = int(input('Informe a Corrente:'))
    print(f'Tensão:{resistencia * corrente}V')

elif opcao == 2:
    tensao = int(input('Informe a Tensão:'))
    corrente = int(input('Informe a Corrente:'))
    print(f'Resistência:{tensao / corrente}Ω')

elif opcao == 3:
    tensao = int(input('Informe a Tensão:'))
    resistencia = int(input('Informe a Resistência:'))
    print(f'Corrente:{tensao / resistencia}A')
else:
    print('How?')

File: Capitulo_3/JavaScript/B-Exercicios-Resolvidos-pag-69-ate-78/exercicio-resolvido-1.js
const prompt = require('prompt-sync')(); 

console.log(`************************************************
        CÁLCULO DE GRANDEZAS ELÉTRICAS
************************************************
1. Tensão (em Volt)
2. Resistência (em Ohm)
3. Corrente (em Ampére)
************************************************`)

let resistencia
let corrente

opcao = Number(prompt('Qual grandeza deseja calcular?'))

switch(opcao){
    case 1:
        resistencia = Number(prompt('Informe a Resistência:'))
        corrente = Number(prompt('Informe a Corrente:'))
        console.log(`Tensão:${resistencia * corrente}V`)
        break;
    case 2:
        tensao = Number(prompt('Informe a Tensão:'))
        corrente = Number(prompt('Informe a Corrente:'))
        console.log(`Resistência:${tensao / corrente}Ω`)
        break;
    case 3:
        tensao = Number(prompt('Informe a Tensão:'))
        resistencia = Number(prompt('Informe a Resistência:'))
        console.log(`Corrente:${tensao / resistencia}A`)
        break;
    default:
        console.log('Opção inválida')
}